.github/workflows/stale.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Close stale PRs'
+on:
+  schedule:
+    - cron: "27 7 * * *"
```

Comment:
What's this time for?

Reply:
The job is set to run at 07:27 every day.
The exact numbers aren't that important here, I just chose some random ones.